src/styles/all.css

```diff
@@ -0,0 +1,944 @@
+.fa {
```

Comment:
Are these icons used in docs homepage from the pro package? Then, I think we should not expose these icons in this public repo. We can load these icons from Lens

Reply:
I tried it without the icons from lens and it didn't load them :/